src/storage/metrics.rs

```diff
@@ -66,60 +68,43 @@ pub fn tls_flush<R: FlowStatsReporter>(reporter: &R) {
 
         reporter.report_read_stats(read_stats);
     });
+    KV_COMMAND_SCAN_DETAILS_STATIC.flush();
 }
 
-pub fn tls_collect_command_count(cmd: &str, priority: CommandPriority) {
-    TLS_STORAGE_METRICS.with(|m| {
-        let mut storage_metrics = m.borrow_mut();
-        storage_metrics
-            .local_kv_command_counter_vec
-            .with_label_values(&[cmd])
-            .inc();
-        storage_metrics
-            .local_sched_commands_pri_counter_vec
-            .with_label_values(&[priority.get_str()])
-            .inc();
-    });
+pub fn tls_collect_command_count(cmd: CommandKind, priority: CommandPriority) {
+    KV_COMMAND_COUNTER_VEC_STATIC.get(cmd).inc();
```

Comment:
I think we can remove these functions and inline the body to the call site. It will be more like its original style :)

Reply:
> I think we can remove these functions and inline the body to the call site. It will be more like its original style :)

In current version of IDEs, a wrap function have a extra benefit, because our struct and methods are generated by macro, which IDE can not well deal with, a wrap function can help us to get better type hint and auto complete features from IDE.